code/game/machinery/pipe/construction.dm

```diff
@@ -238,7 +238,7 @@ Buildable meters
 				return TRUE
 	// no conflicts found
 
-	var/obj/machinery/atmospherics/built_machine = new pipe_type(loc, , , p_init_dir)
+	var/obj/machinery/atmospherics/built_machine = new pipe_type(loc, , fixed_dir(), p_init_dir)
```

Comment:
```suggestion
	var/obj/machinery/atmospherics/built_machine = new pipe_type(loc, null, fixed_dir(), p_init_dir)
```

i dislike leaving empty arguments instead of just writing out `null` so let's fix that here.